docs/source/models/gbd2017/causes/latent_tuberculosis/index.rst

```diff
@@ -4,6 +4,119 @@
 Latent Tuberculosis
 ===================
 
-.. todo:
+TB Overview
+-----------
 
-   all of this
+TB definitions
+++++++++++++++
+Tuberculosis (TB) is a disease caused by the bacteria Mycobacterium
+tuberculosis. It most commonly affects the lungs, although it can affect
+other parts of the body. TB can be fatal if not recognized and treated.
+It also can spread from person to person to infect others. Anyone who 
+inhales the airborne droplets that contains TB bacteria is called a "contact".
+A contact can be someone who living with the person who has infectious TB,
+like a family member, friend, or coworker. [UpToDate]_
+
+What is latent TB?
+++++++++++++++++++
+Latent tuberculosis infection (LTBI) is defined as a state of persistent
+immune response to stimulation by Mycobacterium tuberculosis antigens with
+no evidence of clinically manifest active TB. [WHO]_
+
+What is active TB?
+++++++++++++++++++
+Active tuberculosis, also known as reactivation TB. It occurs when individual's
+immune system becomes weakened and is no longer able to contain the latent
+bacteria; then the TB bacteria become "active", overwhelm the immune system and
+cause a person to become ill. [UpToDate]_ In GBD, active TB consist of 6 child
+causes: Drug-susceptible TB, Multidrug-resistant TB, Extensively drug-resistant
+TB, HIV/AIDS - Drug-susceptible TB, HIV/AIDS - Multidrug-resistant TB, and
+HIV/AIDS - Extensively drug-resistant TB.
```

Comment:
I think it would be helpful to add the abbreviations here, so it's clear what you mean when you use them later. e.g., "Multidrug-resistant TB (MDR-TB)"